hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoGetAccountInfoHandler.java

```diff
@@ -151,7 +151,8 @@ private Optional<AccountInfo> infoForAccount(
         } else {
             final var info = AccountInfo.newBuilder();
             info.ledgerId(ledgerConfig.id());
-            if (!isEmpty(account.key())) info.key(account.key());
+            info.key(account.key());
```

Comment:
This `isEmpty` check should exist. But it should consider `IMMUTABLE_SENTINEL_KEY` imo

Reply:
Mono-service not check this. Here is the [code](https://github.com/hashgraph/hedera-services/blob/27fe1a07dabdf6999015812505acbb78de753032/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/context/primitives/StateView.java#L408) in mono. This fixed one step of the failed HapiTests.

Reply:
hollowAccountCompletionViaNonReqSigIsNotAllowed() and precompileTransferFromHollowAccountWithNeededSigFailsAndDoesNotFinalizeAccount90 failed with:
2023-10-24 17:46:13.513 WARN   262  HapiSpecOperation - 'hollowAccountCompletionViaNonReqSigIsNotAllowed' - CustomSpecAssert failed - java.lang.Exception: Bad account info! :: Bad key! ==> expected: <keyList {
}
> but was: <>!
if we check !isEmpty(account.key() here. Removed this, the tests passed.